### PR TITLE
[PAN-1874] Add badges and missing repo nice-to-have

### PR DESCRIPTION
**Pull request by juanmanuel-tirado**
/

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [x] Test(s)
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

Added some nice-to-have elements for the repo.
- Badges
- Hardened CI images
- Code of conduct moved
- PR template missing
- Missing CODEOWNERS

## Related Tickets & Documents

NA

## QA Instructions

No big changes. Hardened images should be working now.

## [optional] Are there any post deployment tasks we need to perform?

### CODE REVIEW

**Review by chandraprakash — APPROVED** [review]

**Comment by chandraprakash** [comment]
This CODEOWNERS file contains error:
Unknown owner on line 3: make sure the team @pantos-io/management exists, is publicly visible, and has write access to the repository

**Comment by chandraprakash** [comment]
> This CODEOWNERS file contains error: Unknown owner on line 3: make sure the team @pantos-io/management exists, is publicly visible, and has write access to the repository

Looks like its fixed now: This CODEOWNERS file is valid.

**Review by markuslevonyak — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -113,6 +117,7 @@ jobs:
         env:
           REPORT: ${{ steps.slither.outputs.stdout }}
         with:
+          github-token: ${{ secrets.GH_TOKEN }}
```
Comment (jpantos) [inline .github/workflows/ci.yaml:120]: I don't think we need ot use GH_TOKEN, instead we should set
`permissions:
   pull-request: write`

Comment (juanmanuel-tirado) [reply .github/workflows/ci.yaml:120]: permissions are already set

Comment (jpantos) [reply .github/workflows/ci.yaml:120]: Shouldn't in that case we be using secrets.GITHUB_TOKEN? We don't have GH_TOKEN defined

**Review by markuslevonyak — APPROVED** [review]

**Comment by juanmanuel-tirado** [comment]
The results from slither can be sent to the CodeQL summary tab.

**Review by markuslevonyak — APPROVED** [review]

**Review by jpantos — APPROVED** [review]

**Pull request merged**